webapp/app/forms/fields.py

```diff
@@ -199,26 +199,46 @@ def __init__(self, **kwargs):
                                    'data-example-input': _('fields.date_field.example_input.text')}
         super().__init__(**kwargs)
         self.widget = LegacySteuerlotseDateWidget()
+        self.validate_date = validate_date
 
     def _value(self):
         if self.data:
             return [str(self.data.day), str(self.data.month), str(self.data.year)]
         else:
             return self.raw_data if self.raw_data else []
 
+    def process_formdata(self, valuelist):
+        try:
+            super().process_formdata(valuelist)
+        # Prevent standard date validation if validate_date is false.
+        except ValueError as e:
+            if self.validate_date:
+                raise e
+            pass
```

Comment:
Why do we need to do this in the first place? Why shouldn't there be an error if it's not a valid date? Is it because of the custom error message?

Reply:
Ah yes, so it is about the message. Not sure if tis is really that much better but to make it more explicit you could do:
```    
def process_formdata(self, valuelist):
        self._translations._catalog['Not a valid date value'] = _("Not a valid date value.")
        super().process_formdata(valuelist)
```